howso/client/tests/test_client.py

```diff
@@ -354,8 +354,14 @@ def trainee(self, trainee_builder):
             "play": {"type": "nominal"},
         }
         trainee_name = uuid.uuid4().hex
-        trainee = Trainee(trainee_name, features=feats, metadata={'ttl': 600000})
-        trainee_builder.create(trainee)
+        actions = ['play']
+        contexts = ['penguin']
```

Comment:
Couple unused vars

Reply:
Should be addressed by: d66654ea10038f70a275bb4aa504128552745e60